pages/products/_id.vue

```diff
@@ -0,0 +1,355 @@
+<template>
+    <div id="body">
+        <div id="upper">
+            <div id="circles">
+                <span class="dot" id="black"></span>
+                <span class="dot" id="blue"></span>
+                <span class="dot" id="green"></span>
+                <span class="dot" id="skin"></span>
+            </div>
+
+            <img :src="product.image" />
+
+
+
+            <div id="rightpane">
+                <div id="title">
+                    <h1>{{ product.title }}</h1>
+                    <img
+                        src="https://static.vecteezy.com/system/resources/thumbnails/001/505/003/small/share-icon-free-vector.jpg">
+                </div>
+                <p id="TPC">TPC</p>
+                <div id="rating">
+                    <p>4.8<span>&starf;</span>| 15 Ratings</p>
+                    <div class="line"></div>
+                </div>
+
+                <p id="price"><b></b> ${{ product.price }}</p>
+                <p id="choose">Choose a Size</p>
+                <div id="radio">
+                    <p class="Size"><input type="radio" name="Size" class="size" value="Small" id="Small">Small</p>
+                    <p class="Size"><input type="radio" name="Size" class="size" value="Medium" id="Medium">Medium</p>
+                    <p class="Size"><input type="radio" name="Size" class="size" value="Large" id="Large">Large</p>
+                    <p class="Size"><input type="radio" name="Size" class="size" value="Extra Large" id="Extra">Extra Large
+                    </p>
+                    <p class="Size"><input type="radio" name="Size" class="size" value="XXL" id="XXL">XXL</p>
+                </div>
+                <div class="line"></div>
+                <div id="buttons">
+                    <button id="cart"><img src="https://cdn-icons-png.flaticon.com/512/711/711895.png">
+                        <p>Add To Cart</p>
+                    </button>
+                    <button id="buy"><img src="https://icon-library.com/images/current-icon/current-icon-1.jpg">
+                        <p>Buy Now</p>
+                    </button>
+                </div>
+            </div>
+        </div>
+        <div i="image_row">
```

Comment:
hey @rrtk857, You can implement arrows function to show more product images Or you can make issue for the same. For reference :- https://www.youtube.com/watch?v=Y36QpYcnbQY&ab_channel=DennisIvy